### PR TITLE
feat(react): allow props to be passed to next and previous buttons in pointout

### PR DESCRIPTION
**Pull request by scurker**

### CODE REVIEW

**Review by stephenmathieson — COMMENTED** [review]
Change LGTM but I don't understand the motivation behind it. Am also worried about removing the old props. Are we sure they're not in use anywhere?

File: packages/react/src/components/Pointout/index.tsx [hunk 1]
```diff
@@ -24,15 +24,17 @@ export interface PointoutProps {
   ftpoRef: React.Ref<HTMLDivElement>;
   noArrow?: boolean;
   onClose: () => void;
-  onNext: () => void;
```
[the remark this answers is not in the document]
Comment (scurker) [reply packages/react/src/components/Pointout/index.tsx:27]: It's currently an undocumented method and only used by the extension. We also haven't tagged a major version yet.

[the remark this answers is not in the document]
Comment (scurker) [reply packages/react/src/components/Pointout/index.tsx:27]: This is an instance where we're "fixing" something that we haven't published or finalized, so it's technically not breaking.

[the remark this answers is not in the document]
Comment (stephenmathieson) [reply packages/react/src/components/Pointout/index.tsx:27]: Cool OK, thanks for explaining 👍

**Review by stephenmathieson — APPROVED** [review]

**Pull request merged**